armory/instrument/instrument.py

```diff
@@ -311,6 +312,20 @@ def __init__(self):
         self.writers = []
         self.default_writers = []
         self.closed = False
+        self.output_dir = armory.paths.runtime_paths().output_dir
+        self.export_dir = f"{self.output_dir}/saved_samples"
+
+    def _set_output_dir(self, name):
+        self.output_dir = name
+
+    def get_output_dir(self):
+        return self.output_dir
+
+    def _set_export_dir(self, name):
+        self.export_dir = name
+
+    def get_export_dir(self):
+        return self.export_dir
```

Comment:
Does it make more sense to enforce export_dir to be a subdir of output_dir? Otherwise, if you're in the container, you're likely going to make an error in pathing that doesn't get saved.

Then this section would become:
```
        self._set_export_dir("saved_samples")

    def _set_export_dir(self, output_subdir):
        self.export_dir = os.path.join(armory.paths.runtime_paths().output_dir, output_subdir)

    def get_export_dir(self):
        return self.export_dir
```

Reply:
just thinking about whether there could be a situation where you want to export artifacts to somewhere other than `~/.armory/outputs`...but I think it's cleanest to have everything in one place so I like the change.

I do think the logic of setting export_dir to `f"saved_samples_{self.time_stamp}"` when `"saved_samples"` already exists needs to remain, so that outputs from subsequent runs don't get jumbled up / overwrite eachother

Reply:
Yeah, I agree.